### PR TITLE
[SMALLFIX] Remove unintended trace

### PR DESCRIPTION
**Pull request by jiacheliu3**
### What changes are proposed in this pull request?

Remove one log entry which was accidentally brought in from testing

### CODE REVIEW

**Review by elega — APPROVED** [review]

File: tests/src/test/java/alluxio/server/worker/WorkerAllMasterRegistrationTest.java [hunk 1]
```diff
@@ -80,7 +80,6 @@ public void before() throws Exception {
     Configuration.set(PropertyKey.MASTER_EMBEDDED_JOURNAL_WRITE_TIMEOUT, "10sec");
     Configuration.set(PropertyKey.MASTER_EMBEDDED_JOURNAL_MIN_ELECTION_TIMEOUT, "3s");
     Configuration.set(PropertyKey.MASTER_EMBEDDED_JOURNAL_MAX_ELECTION_TIMEOUT, "6s");
-    Configuration.set(PropertyKey.WORKER_REGISTER_TO_ALL_MASTERS, true);
```
Comment (jiacheliu3) [inline tests/src/test/java/alluxio/server/worker/WorkerAllMasterRegistrationTest.java:83]: this line is dup to L75

File: tests/src/test/java/alluxio/server/worker/WorkerAllMasterRegistrationTest.java [hunk 2]
```diff
@@ -70,7 +71,6 @@ public void before() throws Exception {
         mNumMasters, mNumWorkers, PortCoordination.WORKER_ALL_MASTER_REGISTRATION);
     mCluster.initConfiguration(
         IntegrationTestUtils.getTestName(getClass().getSimpleName(), mTestName.getMethodName()));
-    Configuration.set(PropertyKey.MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES, 5);
```
Comment (jiacheliu3) [inline tests/src/test/java/alluxio/server/worker/WorkerAllMasterRegistrationTest.java:73]: this line makes the master checkpoint more than it should be, and checkpointing logic is irrelevant to the tests here

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.10 please

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.10 please

**Pull request merged**